app/views/ingredients/_form.html.erb

```diff
@@ -24,7 +24,7 @@
       <div class="field">
         <div class="control">
           <div class="select is-primary">
-            <%= form.select :category, Ingredient::CATEGORIES %>
+            <%= form.select :category, IngredientCategory.pluck("name").sort, prompt: "Choose a category" %>
```

Comment:
ingredients category should probably be declared in controller
```
@ingredients_categories - IngredientCategory.pluck("name").sort
```

Reply:
Fixed this using a constant within the ingredient.rb model